### ISSUE TITLE
Still working?

### ISSUE THREAD

**Issue by ArnoutVerbeken** [bollewolle/delijn-sensor#1]
Hello,

I'm trying to set this up, but it does not seem to work anymore...
I made all changes (move to custom_components/delijn/delijn.py) and also made a manifest.json file, but I always get the error:
Platform not found: sensor.delijn

Do you still use this?  Should this still be working?

Thanks for your efforts!

**Comment by bollewolle** [comment]
I've actually did a Pull Request to have it added into the core of Home Assistant itself, but it's still under review at the moment.

I don't have my dev pc with me at the moment but iirc you also need to add an __init__.py now for the component to be picked up. You can see the code from the PR I've send to Home Assistant here: https://github.com/bollewolle/home-assistant/tree/delijn/homeassistant/components/delijn

Adding that to the custom_components/delijn folder should work, but I'm not 100% sure to be honest.

In any case, once it's in HA itself I will remove this repository as it won't be needed anymore to add a custom_component.

Hope this helped a bit.

Cheers!

**Comment by ArnoutVerbeken** [comment]
Thanks, working fine with the components from the link above.

I think however I already found a bug...
For some fields, the hour is 1hour off.
E.g:
Current time : 22h30
Due in: 9 minutes
Due at (realtime): 23h39
Due at (scheduled:23h40

You see that for the "Due at" fields, the time is off for 1 hour.  Testes with different "haltes".

Hope it helps.

Update:
It seems that the sensor itself is fine and gives the correct time, but that your lovelace card suffers from the above issue....  Must be some timing setting somewhere? (summertime?)

Update:  solved the lovelacard by changing line 168 from
var dt = new Date(newText + "+01:00");
to
var dt = new Date(newText);

Don't know what you were trying to achieve here?  Seems like the sensortime itself is always correct so I don't see the need to add 1 hour....

**Comment by bollewolle** [comment]
Yeah, there were some issues and "hard coded logic" which wasn't very nice as a solution.

I've done an update on the code for the sensor itself (both in the repo as towards Home-Assistant) and to the pydelijn package so now it will use the UTC timestamps instead of the local times that the De Lijn API sends back. After that I did an update of the above code you mentioned and just pushed it towards this repo. So now it should all work again.
